Library/Homebrew/dev-cmd/pr-pull.rb

```diff
@@ -291,34 +302,49 @@ def cherry_pick_pr!(user, repo, pr, args:, path: ".")
     Utils::Git.cherry_pick!(path, "--ff", "--allow-empty", *commits, verbose: args.verbose?, resolve: args.resolve?)
   end
 
-  def formulae_need_bottles?(tap, original_commit, user, repo, pr, args:)
+  def packages_need_bottles?(tap, original_commit, user, repo, pr, args:)
     return if args.dry_run?
 
     labels = GitHub.pull_request_labels(user, repo, pr)
 
     return false if labels.include?("CI-syntax-only") || labels.include?("CI-no-bottles")
 
-    changed_formulae(tap, original_commit).any? do |f|
-      !f.bottle_unneeded? && !f.bottle_disabled?
+    changed_packages(tap, original_commit).any? do |f|
+      !f.instance_of?(Cask::Cask) && !f.bottle_unneeded? && !f.bottle_disabled?
     end
   end
 
-  def changed_formulae(tap, original_commit)
+  def changed_packages(tap, original_commit)
     if Homebrew::EnvConfig.disable_load_formula?
       opoo "Can't check if updated bottles are necessary as HOMEBREW_DISABLE_LOAD_FORMULA is set!"
       return
     end
```

Comment:
This probably shouldn't trigger for casks?

Reply:
I'm not sure what should happen to casks in this case, what would normal cask operations do with `HOMEBREW_DISABLE_LOAD_FORMULA`?

Reply:
Should just add a check to this conditional to ensure it doesn't fail for casks IMO